src/main/resources/assets/alexscaves/lang/zh_cn.json

```diff
@@ -264,305 +264,306 @@
   "item.alexscaves.primordial_pants": "原始裤子",
-  "item.alexscaves.dinosaur_pottery_sherd": "恐龙陶罐碎片",
-  "item.alexscaves.footprint_pottery_sherd": "脚印陶罐碎片",
+  "item.alexscaves.dinosaur_pottery_sherd": "恐龙纹样陶片",
+  "item.alexscaves.footprint_pottery_sherd": "足迹纹样陶片",
   "item.alexscaves.dinosaur_train": "恐龙火车",
   "item.alexscaves.acid_bucket": "酸液桶",
-  "item.alexscaves.radgill_bucket": "辐鳃鱼桶",
-  "item.alexscaves.radgill": "辐鳃鱼",
-  "item.alexscaves.cooked_radgill": "熟辐鳃鱼",
-  "item.alexscaves.sulfur_dust": "硫粉",
-  "item.alexscaves.uranium": "铀晶",
-  "item.alexscaves.uranium_shard": "铀晶碎片",
+  "item.alexscaves.radgill_bucket": "辐射鳃鱼桶",
+  "item.alexscaves.radgill": "辐射鳃鱼",
+  "item.alexscaves.cooked_radgill": "熟辐射鳃鱼",
+  "item.alexscaves.sulfur_dust": "硫磺粉",
+  "item.alexscaves.uranium": "铀",
+  "item.alexscaves.uranium_shard": "铀碎片",
   "item.alexscaves.radon_bottle": "氡气瓶",
-  "item.alexscaves.cinder_brick": "熔渣砖",
-  "item.alexscaves.spelunkie": "硫心糖果",
-  "item.alexscaves.slam": "猛汉猪排",
+  "item.alexscaves.cinder_brick": "煤灰石砖",
+  "item.alexscaves.spelunkie": "探洞夹心蛋糕",
+  "item.alexscaves.slam": "满腹午餐肉",
   "item.alexscaves.green_soylent": "“绿色食品”",
-  "item.alexscaves.toxic_paste": "毒化烂泥",
+  "item.alexscaves.toxic_paste": "剧毒膏",
   "item.alexscaves.polymer_plate": "聚合物板",
   "item.alexscaves.fissile_core": "裂变核心",
-  "item.alexscaves.charred_remnant": "焦化残骸",
+  "item.alexscaves.charred_remnant": "烧焦的核熔炉残骸",
   "item.alexscaves.hazmat_mask": "防化面罩",
   "item.alexscaves.hazmat_chestplate": "防化胸甲",
   "item.alexscaves.hazmat_leggings": "防化护腿",
   "item.alexscaves.hazmat_boots": "防化靴子",
   "item.alexscaves.remote_detonator": "遥控起爆器",
-  "item.alexscaves.remote_detonator.desc": "绑定至(%s, %s, %s)",
+  "item.alexscaves.remote_detonator.desc": "绑定到炸弹的位置 (%s, %s, %s)",
   "item.alexscaves.raygun": "镭射枪",
-  "item.alexscaves.raygun.charge": "充能：%s/%s",
-  "item.alexscaves.lanternfish_bucket": "灯笼鱼桶",
-  "item.alexscaves.lanternfish": "灯笼鱼",
-  "item.alexscaves.cooked_lanternfish": "熟灯笼鱼",
-  "item.alexscaves.tripodfish_bucket": "三脚鱼桶",
-  "item.alexscaves.tripodfish": "三脚鱼",
-  "item.alexscaves.cooked_tripodfish": "熟三脚鱼",
+  "item.alexscaves.raygun.charge": "能量剩余: %s/%s",
+  "item.alexscaves.lanternfish_bucket": "灯鱼桶",
+  "item.alexscaves.lanternfish": "灯鱼",
+  "item.alexscaves.cooked_lanternfish": "熟的灯鱼",
+  "item.alexscaves.tripodfish_bucket": "三足鱼桶",
+  "item.alexscaves.tripodfish": "三足鱼",
+  "item.alexscaves.cooked_tripodfish": "熟三足鱼",
   "item.alexscaves.sea_pig_bucket": "海猪桶",
   "item.alexscaves.sea_pig": "海猪",
-  "item.alexscaves.marine_snow": "海雪",
-  "item.alexscaves.gossamer_worm_bucket": "浮蚕桶",
-  "item.alexscaves.bioluminesscence": "生物荧光质",
-  "item.alexscaves.cooked_mussel": "熟贻贝",
-  "item.alexscaves.deep_sea_sushi_roll": "深海寿司卷",
-  "item.alexscaves.sea_glass_shards": "海洋玻璃碎片",
-  "item.alexscaves.submarine": "潜水艇",
+  "item.alexscaves.marine_snow": "深海雪",
+  "item.alexscaves.gossamer_worm_bucket": "薄纱海蠕虫",

```

Comment:
**Inappropriate translation.** This text means "Sea worm with thin silks", which doesn't match with the original text. 
```suggestion
  "item.alexscaves.gossamer_worm_bucket": "浮蚕桶",
```